keras_cv/layers/object_detection/multi_class_non_max_suppression_test.py

```diff
@@ -39,3 +41,60 @@ def test_decode_predictions_output_shapes(self):
         self.assertEqual(result["boxes"].shape, [8, 100, 4])
         self.assertEqual(result["classes"].shape, [8, 100])
         self.assertEqual(result["confidence"].shape, [8, 100])
+
+
+class XlaMlirBridgeNmsPredictionDecoderTest(tf.test.TestCase):
+    tf.config.experimental.enable_mlir_bridge()
```

Comment:
@bhack is this more similar to TPU compilation than normal XLA?  (Just curious)

Reply:
No, this about forcing to use the new MLIR bridge without fallback on the old one. 
I guess that almost all the development resources are landing on the new MLIR bridge instead of the old one. 

It is why sometimes we could pass the compilation with the new bridge but not with the old one.
Also I suppose that many type of BUG/FR in TF are not going to receive too much attention if submitted on the old bridge (but I could be wrong here).

As you can see TF nightly removed the [experimental status of this bridge](https://www.tensorflow.org/api_docs/python/tf/config/experimental/enable_mlir_bridge?version=nightly) but it is still not fully enabled by default.

Reply:
Instead for an improved feedback to developers on what it is supported and what not when we are going to compile a function see our threads with @paynecl at:
https://github.com/openxla/stablehlo/issues/216#issuecomment-1263615220
https://github.com/tensorflow/tensorflow/pull/56510